### DISCUSSION TITLE
model up the map

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#5070] [Q&A]
now,i change the revit to bim,and use itwin open the bim file
in revit file ,the height for PBP,the height for SP, is 0
but in imodel show height is 54.87m and the all element of model on the map 
( the element  under the PBP point must under the map ) 
and now how set the   element  under the PBP point  under the map

revit verison :2022

![(X9R{~MKVF0YHEQBD_3B)RN](https://user-images.githubusercontent.com/18022720/218711904-cc7b1b0a-57bb-464c-9680-f96e7d287f68.png)

![VJCGK30957M23I8D Z~B`ZV](https://user-images.githubusercontent.com/18022720/218712317-dfecf0cd-0808-449a-aa91-5056a2c005e8.png)

![_W@ 9I6B470C`)H 0( @U`E](https://user-images.githubusercontent.com/18022720/218711948-cf6e908e-6e7e-4d92-911e-2db73a6fb7d4.png)

**Comment by timlawrence-bentley** [comment]
Hi,
One of our Revit connector developers looked at this and can't give an answer based on the info provided. Could you please either submit a service request here: https://apps.bentley.com/srmanager/ProductSupport or if that doesn't work, please email me tim.lawrence@bentley.com and I'll give you a spot to upload the model. We'd also need access to look at your iModel to see what exists already.

Thanks,

Tim
